src/doc/rustdoc/src/what-is-rustdoc.md

```diff
@@ -70,16 +73,17 @@ check it out! It should show a link to the `foo` function's page, which
 is located at `doc/docs/fn.foo.html`. On that page, you'll see the "foo is
 a function" we put inside the documentation comment in our crate.
 
-## Using rustdoc with Cargo
+## <a name="rustdoc-cargo"></a> Using rustdoc with Cargo
```

Comment:
Please don't add HTML tags (it's also not how you create an anchor). An anchor already exists in any case.